lib/eco/incparser/incparser.py

```diff
@@ -385,6 +400,9 @@ def pass2(self, node, offset, error_offset):
             offset += c.textlength()
 
     def is_retainable_subtree(self, node):
+        # Subtrees are currently always unretainable due to a potential bug in
+        # Wagners algorithm
+        return False
         if node.new:
```

Comment:
Yes. I just don't want to delete that code because the implementation is correct. So once I found out what the problem is with the retainability algorithm is I just have to remove the `return False`.